website/templates/project/project.mako

```diff
@@ -256,11 +256,11 @@
             <div class="panel-body" style="display:none">
                 <dl id="citationList" class="citation-list">
                     <dt>APA</dt>
-                        <dd class="citation-text" data-bind="text: apa"></dd>
+                        <dd class="citation-text" data-bind="html: apa"></dd>
```

Comment:
Vigorously agreed. The only way this would be ok is if all data sent to citeproc was first ensured to be properly escaped before passing to citeproc for construction of the formatted HTML string.

I'll need to dig into this more- can you be sure to assign me when CR2 is ready? And Nan, I'd be happy to walk through this with you as part of the review process, especially if you have any insights into how citation formatting is done.